src/RestApiClient.js

```diff
@@ -77,7 +77,10 @@ class RestApiClient {
    */
   get(path, query = {}) {
     const url = new URL(path, this.baseUrl);
-    url.search = stringifyQuery(query);
+
+    if (query && typeof query === 'object') {
+      url.search = stringifyQuery(query);
+    }
 
```

Comment:
@chloealee I think you still need to add the `query` to the `url.search` property right? Looks like you _only_ add it to the `url.search` if it gets stringified, but if it's already a string you're not adding it to the `url`.

Reply:
@weerd yes bingo!! thank you! I'll do another PR for that now!